javascripts/discourse/components/featured-homepage-topics.hbs

```diff
@@ -1,12 +1,23 @@
-{{#if showFor}}
-  {{#if showHere}}
-    {{#if featuredTagTopics}}
-      <div class="custom-homepage-wrapper">
+{{#if this.showFor}}
+  {{#if this.showHere}}
```

Comment:
Could merge those to decrease overall indentation in the file

`{{#if (and this.showFor this.showHere)` or have a new getter (e.g. `get visible()`) that does an `&&` on those.